src/import/FSHImporter.ts

```diff
@@ -2057,13 +2057,10 @@ export class FSHImporter extends FSHVisitor {
     }
 
     // Replace '[+]' with '[=]' in the version of the path to add to pathContext, unless the rule is only used to set a path
```

Comment:
Should be fixed by https://github.com/FHIR/sushi/pull/907/commits/60dec8510003dcf32a9ec0010e56f0e0704149bd. I also added a more explicit test since figured that may be a good idea.